src/screens/index.ts

```diff
@@ -0,0 +1 @@
+export { default as App } from './App';
```

Comment:
Remove old architecture for study change it how it need. For now, why you do it ?